test/automated/reflection/actuator/actuator.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../../automated_init'
+
+context "Reflect" do
+  context "Reflection" do
+    context "Actuator" do
+      constant_name = Controls::Namespace::Random.get
+      subject = Controls::Subject.example
+      target = Controls::Target.example(constant_name:, subject:)
+
+      method_name = :some_method
+      RecordInvocation.(target, method_name)
+
+      reflection = Reflect.(subject, constant_name)
+      reflection.(method_name, subject, :some_arg)
```

Comment:
Consider: `reflect` as the variable name. Rationale: `some_noun.()` doesn't make sense but `some_verb.()` does. (`some_noun.some_verb` also makes sense but isn't relevant here)